General/CodeSmells/Comments/Src/Comments.Problem/TooManyComments/Good/Person.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Comments.Problem.TooManyComments.Good
+{
+    public class Person
+    {
+        public string Name { get; }
+        public DateTime Birthday { get; }
+        // If objects can manage their own state without outside intervention- they should.
+        // Person has enough info to calculate their own age.
+        public int Age => (int)(DateTime.Now - Birthday).TotalDays / 365;
```

Comment:
public int Age => (int)Math.Floor((DateTime.Now - Birthday).TotalDays / 365.25);// Because leap years.

Reply:
Holy crap 🗡 Will change

Reply:
Changed to include leap years.